test/run-test.ps1

```diff
@@ -51,6 +52,7 @@ Try {
     }
 
     $env:IMAGE_ARCH_FILTER = $Architecture
```

Comment:
Are these alphabetical as in `Architecture`, `OS` and `Version`?

Reply:
These sorted by priority.  Which is most likely to be used.